ush/config.sh

```diff
@@ -1,41 +1,38 @@
 MACHINE="hera"
```

Comment:
The config.sh file should not be committed to the repository. If changes are made to config.community or config.nco, those should be committed.